### PR TITLE
Start update to Next.js 6

### PR DESCRIPTION
**Pull request by rogerhutchings**
Package: `app-project`

Creates a new base `app-project` based on Next.js 6. Finally got the test harness working 👍 So my plan is to get this reviewed and merged, and then do another PR copying the existing code from `app-project` into `app-project-2`, and then delete the old app and rename this one.

# Review Checklist

## General

- [ ] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?
- [ ] Is the changelog updated?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `rm -rf node_modules/ && npm install` and app works as expected?

### CODE REVIEW

File: packages/app-project-2/server.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      if (!params.subroute) {
+        return renderPage('/Home')
+      }
+
+      if (params.subroute.includes('classify')) {
+        return renderPage('/Classify')
+      }
+
+      if (params.subroute.includes('about')) {
+        return renderPage('/About')
```
Comment (srallen) [inline packages/app-project-2/server.js]: There's inconsistent use of case in the actual file names versus what is being passed as params for each `renderPage` call here. The files should match the strings being provided. Right now I get a 404 running the app in dev mode going to the index page. If I fix the file name to be `Index.js`, the route matches correctly.

**Comment by rogerhutchings** [comment]
Cool. There's also something pretty annoying where it appears `styled-components` is being loaded multiple times, which is borking styling (_sometimes_). I'll fix these, and I want to get that sorted before this merges.

**Comment by rogerhutchings** [comment]
I've fixed those issues, and bumped to Next 7 since that's come out(!)

**Comment by srallen** [comment]
I'm still getting 404s for all of the pages. I see you disabled file system routing, but if I'm reading that correctly, then client side routing has to be setup instead?

**Comment by rogerhutchings** [comment]
That should be set from https://github.com/zooniverse/front-end-monorepo/pull/137/files#diff-145fb190e8570c05577e8b78bc24b96eR28 onwards - I'll check it again...

**Comment by rogerhutchings** [comment]
TIL OSX is both case insensitive and case preserving 🤦‍♂️ I've `git mv`'d those files and they should work now.

**Pull request merged**